src/h3/api/_api_template.py

```diff
@@ -136,24 +178,65 @@ def h3_distance(h1, h2):
         return d
 
     def h3_to_geo_boundary(h, geo_json=False):
+        """ Return tuple of lat/lng paris describing
+        the cell boundary
+
+        If `geo_json` is True, return lng/lat pairs
+        and have the last pair be the same as the first.
+
+        Returns
+        -------
+        Tuple[(Float, Float)]
+
+        """
         return _cy.cell_boundary(_in_scalar(h), geo_json)
 
     def k_ring(h, k=1):
+        """ Return unordered set of cells with H3 distance
+        `<= k` from `h`. "Full" disk.
+
+        todo: how to describe different output collection format for different
+        interfaces?
+
+        Returns
+        -------
+        Set[H3 Index]
+        """
         mv = _cy.disk(_in_scalar(h), k)
 
         return _out_unordered(mv)
 
     def hex_range(h, k=1):
+        """ Alias for `k_ring`. "Full" disk.
```

Comment:
Note that this differs from the C API - hexRange is a different function there.

Reply:
note added